example/docker-compose.extra.yml

```diff
@@ -0,0 +1,102 @@
+# Wikibase for testing
+version: '3'
+
+services:
+  elasticsearch:
+    image: "${ELASTICSEARCH_IMAGE_NAME}"
+    restart: unless-stopped
+    networks:
+      default:
+        aliases:
+         - elasticsearch.svc
+    environment:
+      discovery.type: single-node
+      ES_JAVA_OPTS: "-Xms512m -Xmx512m"
+  wdqs-frontend:
+    image: "${QUERYSERVICE_UI_IMAGE_NAME}"
+    restart: unless-stopped
+    ports:
+     - "${WDQS_FRONTEND_PORT}:80"
+    depends_on:
+      - wdqs-proxy
+    networks:
+      default:
+        aliases:
+         - wdqs-frontend.svc
+    environment:
+      - WIKIBASE_HOST=${WIKIBASE_HOST}
+      - WDQS_HOST=wdqs-proxy.svc
+  wdqs:
+    image: "${QUERYSERVICE_IMAGE_NAME}"
+    restart: unless-stopped
+    command: /runBlazegraph.sh
+    volumes:
+      - query-service-data:/wdqs/data
+    networks:
+      default:
+        aliases:
+         - wdqs.svc
+    environment:
+      - WIKIBASE_HOST=${WIKIBASE_HOST}
+      - WDQS_HOST=wdqs.svc
+      - WDQS_PORT=9999
+    expose:
+      - 9999
+  wdqs-proxy:
+    image: "${WDQS_PROXY_IMAGE_NAME}"
+    restart: unless-stopped
+    environment:
+      - PROXY_PASS_HOST=wdqs.svc:9999
+    depends_on:
+      - wdqs
+    ports:
+      - "8989:80"
```

Comment:
move to env!

Reply:
remove! doesn't even make sense why should the proxy be public when the traffic goes through docker?